github/event_types.go

```diff
@@ -653,6 +675,21 @@ type StatusEvent struct {
 	Installation *Installation     `json:"installation,omitempty"`
 }
 
+// TeamEvent is triggered when an organization's team is created, modified or deleted.
+// The Webhook event name is "team".
+//
+// Events of this type are not visible in timelines. These events are only used
+// to trigger hooks.
+//
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#teamevent
+type TeamEvent struct {
+	Action  *string       `json:"action,omitempty"`
+	Team    *Team         `json:"team,omitempty"`
+	Changes *TeamChange   `json:"changes,omitempty"`
+	Org     *Organization `json:"organization,omitempty"`
+	Sender  *User         `json:"sender,omitempty"`
```

Comment:
It looks like you're missing ``Repo *Repository `json:"repository,omitempty"` `` struct field. I see it's a part of Events API payload at https://developer.github.com/v3/activity/events/types/#events-api-payload-34:

![image](https://cloud.githubusercontent.com/assets/1924134/26736705/469860c8-4795-11e7-82f6-26bd5b8bb1fd.png)

Note this is different from the `repository` field inside `changes` field.